allennlp/data/fields/label_field.py

```diff
@@ -57,12 +57,12 @@ def __init__(self,
     @overrides
     def count_vocab_items(self, counter: Dict[str, Dict[str, int]]):
         if self._label_id is None:
-            counter[self._label_namespace][self._label] += 1
+            counter[self._label_namespace][self._label] += 1  # type: ignore
```

Comment:
`Invalid index type "Union[str, int]" for Dict[str, int]; expected type "str"`

because `self.label` is technically `Union[str, int]` and the key needs to be just str. I think this could be fixed with a `cast`, maybe I'll do that in the future (but I don't like using `cast`)